Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -182,6 +306,44 @@ String CSSCounterStyle::initialRepresentation(int value) const
         return counterForSystemAdditive(absoluteValue);
     case CSSCounterStyleDescriptors::System::Fixed:
         return counterForSystemFixed(value);
+    case CSSCounterStyleDescriptors::System::SimplifiedChineseInformal:
+        static constexpr UChar simplifiedChineseInformalTable[17] = {
```

Comment:
Can we share these tables with `RenderListMarker.cpp`? Seems unnecessary to have two sets.

Reply:
Sharing is not trivial, and the goal is to eventually remove the code in `RenderListMarker.cpp` (which is only exercised when CSSCounterStyleAtRulesEnabled is off), so I'd rather focus on that goal instead (which this patch gets us closer to!)

Reply:
Actually it shouldn't be too hard to share the tables, taking care of it.